mitiq/interface/conversions.py

```diff
@@ -108,25 +99,24 @@ def convert_to_mitiq(circuit: QPROGRAM) -> Tuple[Circuit, str]:
 
         input_circuit_type = "pennylane"
         conversion_function = from_pennylane
+
+    elif register_from_dict.get(package):
+        input_circuit_type = package
+        conversion_function = register_from_dict.get(package)
```

Comment:
Oh shoot I've never had a chance to actually use the [walrus operator](https://realpython.com/python-walrus-operator/) (`:=`), but I think this might be a good use case.
```suggestion
    elif (conversion_function := register_from_dict.get(package)):
        input_circuit_type = package
```

Reply:
@natestemen tests pass with the walrus, fun stuff! But I'm not sure how to resolve the type errors now, so if we could brainstorm for a few that'd be great!